app/admin/RTMediaAdmin.php

```diff
@@ -963,10 +956,21 @@ public function ui( $hook ) {
 					'request_failed'                => esc_html__( 'Request failed.', 'buddypress-media' ),
 					'wrong_css_input'               => esc_html__( 'You can not use @import statement in custom css', 'buddypress-media' ),
 				);
-
 				wp_localize_script( 'rtmedia-admin', 'rtmedia_admin_strings', $rtmedia_admin_strings );
-				wp_localize_script( 'rtmedia-admin', 'settings_url', esc_url( add_query_arg( array( 'page' => 'rtmedia-settings' ), ( is_multisite() ? network_admin_url( 'admin.php' ) : admin_url( 'admin.php' ) ) ) ) . '#privacy_enabled' );
-				wp_localize_script( 'rtmedia-admin', 'settings_rt_album_import_url', esc_url( add_query_arg( array( 'page' => 'rtmedia-settings' ), ( is_multisite() ? network_admin_url( 'admin.php' ) : admin_url( 'admin.php' ) ) ) ) );
+
+				wp_localize_script(
+					'rtmedia-admin',
+					'rtmedia',
+					array(
+						'on_label'                     => __( 'ON', 'buddypress-media' ),
+						'off_label'                    => __( 'OFF', 'buddypress-media' ),
+						'admin_ajax'                   => $admin_ajax,
+						'admin_url'                    => admin_url(),
+						'fileupload_url'               => RTMEDIA_URL . 'app/helper/rtUploadAttachment.php', /* path for file upload using ajax */
+						'settings_url'                 => esc_url( add_query_arg( array( 'page' => 'rtmedia-settings' ), ( is_multisite() ? network_admin_url( 'admin.php' ) : admin_url( 'admin.php' ) ) ) ) . '#privacy_enabled',
+						'settings_rt_album_import_url' => esc_url( add_query_arg( array( 'page' => 'rtmedia-settings' ), ( is_multisite() ? network_admin_url( 'admin.php' ) : admin_url( 'admin.php' ) ) ) ),
+					)
+				);
```

Comment:
I don't see any changes made in settings script and so it's giving console errors.

Also if the var names has changed here, does it work with all add-ons? If it doesn't work please do the same as it's done for frontend scripts.